### DISCUSSION TITLE
A summary of the history from the beginning to today of the ADempiere project 2006 to 2023

### DISCUSSION THREAD

**Discussion by e-Evolution** [adempiere/adempiere#4133] [General]
A summary of the history from the beginning to today of the ADempiere project

2006 is created ADempiere :

## The story here :

https://wiki.adempiere.net/History_of_ADempiere

## A Council is formed to govern the project

- A Project Council is formed
https://wiki.adempiere.net/Council

- The Council define el Project_Charter
https://wiki.adempiere.net/Project_Charter

- The principles are established.
https://wiki.adempiere.net/Project_Charter#Adempiere_Principles

## The project structure is defined
https://wiki.adempiere.net/Project_Charter#Political

- Is defined  and setting the PMC (Project Management Committee) and CC (Commit Committee)
https://wiki.adempiere.net/Project_Management_Committee
https://wiki.adempiere.net/Commit_Committee

- The structure of the PMC and the concept of its Head are established
https://wiki.adempiere.net/PMC_Head_Concept
https://wiki.adempiere.net/PMC_Structure

## The ADempiere foundation is born

Three years later, the creation of a foundation for the protection of assets and brand management for the ADempiere project is discussed at the second world conference
https://wiki.adempiere.net/European_ADempiere_Conference_Day2#Adempiere_Foundation
https://wiki.adempiere.net/Foundation_Project

## The new ABC association is created
In the same 2009 world conference, leading consulting companies worldwide propose the creation of an association of consulting companies to standardize implementation methodologies, share use cases, improve the business proposal, offer professional support for ADempiere under the domain adempiereconsultants ( ABC) with domain http://www.adempiereconsultants.com (the ABC domain)

Akuna Inc, England, CEO Michael Judd, https://wiki.adempiere.net/User:Juddm
Systemhaus Westfalia, El Salvador, CEO Mario Calderon,https://wiki.adempiere.net/User:Mar_cal_westf
Catura AG, Germany, CEO Werner Scharinger and Thomas Kresner
Catura AG, Bulgaria, CTO Trifon https://wiki.adempiere.net/User:Trifonnt
Freelance , Romania , CTO Teo Scarca , now of Metasfresh https://wiki.adempiere.net/User:Teo_sarca
e-Evolution, México, CE Victor Perez https://wiki.adempiere.net/User:Vpj-cd

Days later other companies join the ABC initiative

OFB Consulting Ltda , Chile , Ramiro Vergara , CEO https://wiki.adempiere.net/User:Rvergara
Nickel Networks Inc, USA, Paul Aviles, CEO , 

Ramiro Vergara, Thomas Kresner ,  spokesperson announces ABC to the world

https://sourceforge.net/p/adempiere/discussion/611167/thread/bf89a4e3/
https://www.prlog.org/10693986-adempiere-implementers-form-an-alliance-for-worldwide-service-offering.html
https://www.openpr.com/news/132620/adempiere-implementers-form-an-alliance-for-worldwide-service-offering.html
https://www.heise.de/news/Support-fuer-ERP-Loesung-Adempiere-1008018.html

## ABC begins to collaborate as a block in the release
https://wiki.adempiere.net/Release_360LTS , see the signatories with ABC

## Staring the civil war between the years 2010-2011

The PMC questions the formation of ABC alleging that as it is made up of consulting companies they can interfere in the project and the power of PMC is diminished

Given the creation of ABC, the discussion of the use of brands and the ownership of the project's assets began.

## The ADempiere German foundation becomes the ADempiere Foundation for the whole world
As a result, the German Foundation is proposed to become a worldwide entity, which was the organization that had the best structure and being non-profit to protect the brand rights and assets of the project.

https://wiki.adempiere.net/ADeV

## The rules of use of the brand are established.
https://wiki.adempiere.net/Brand_Usage_Rules

##  The German ADempiere foundation becomes the ADempiere  Foundation for the whole world

A vote is taken and by a majority of the citizens, the German foundation Adempiere eV becomes a World Foundation ADempiere Foundation and custody of the property rights and assets of the project is handed over

This was decided via citizens vote on 07th of March 2011.

Results are published

https://sourceforge.net/p/adempiere/discussion/611167/thread/7f67cc69/

## The ADempiere foundation  claims the assets

The foundation requests that all those who have domains around Adempiere have turned themselves in to the foundation

## The Big Discussion

https://sourceforge.net/p/adempiere/discussion/611167/thread/8550ce35/

www.adempiereconsultants.com ABC decides to dissolve in favor of the foundation, complying with the mandate of the citizens and hands over the domain in favor of the Foundation

https://sourceforge.net/p/adempiere/discussion/611167/thread/1a28e6f1/

The PMC headed by CR and the self-appointed benevolent dictator Red1 disregard and disobey the decision of the citizens that was voted and never delivers the domains adempiere.com and adempiere.org

CR , like a classic banana dictator calls itself the rightful protector of the community and  funded by Trek Global https://www.idempiere.org/organizations/  initiates a fork of ADempiere, elsewhere to the detriment of the project, then CR is kicked out of the project

## Dissolution of the PMC and Council
To prevent power from being controlled by a single person, the PMC, CC are dissolved and the functional and technical teams are formed with the aim of simplifying the contribution process through the development procedure.https://wiki.adempiere.net/Software_Development_Procedure , backed by citizens and the ADempiere foundation

This concept was accepted by the citizen by vote on 29th of November 2010.

The Functional team is formedhttps://wiki.adempiere.net/Functional_Team
The Technical Team is formedhttps://wiki.adempiere.net/Technical_Team

## Later important events:

Norbert, Kai, Mark resign from the ADempiere Foundation

Metas and its CEO Norbert , CTO Mark the main implementer in Germany leave the project and create a fork called Metasfresh https://github.com/metasfresh/metasfresh based on ADempiere

Steven Sackett resigns from the foundation 

Adaxa Suite is born as a friendly fork based on ADempiere https://en.wikipedia.org/wiki/Adaxa_Suite

## A new board is formed 

Michael Mackey , Michael Judd , Mario Calderon , Susanne Calderón , Víctor Pérez make up the following board directory

Michael Mackey resigns from the foundation to join metasfresh https://metasfresh.com/en/find-a-partner/

Yamel Senih for their great contributions to the project  is invited by the Board of Directors and joins the board directory August 2, 2023

## Current ADempiere Foundation Board directory

Victor Perez
Michel Judd 
Mario cauldron
Sussane Calderon
Yamel Senih

## Ten years of hard work and collaboration

Since the dissolution of the Council and PMC, after a long period of calm and hard work, the ADempiere Foundation, the Functional Team and the Technical Team have continuously released 7 versions, thus guaranteeing the continuity of the project.

Fulfill 370 https://github.com/adempiere/adempiere/releases/tag/3.7.0
Fulfill 380 https://github.com/adempiere/adempiere/releases/tag/3.8.0
Fulfill 390 https://github.com/adempiere/adempiere/releases/tag/3.9.0
Fulfill 391 https://github.com/adempiere/adempiere/releases/tag/3.9.1
Fulfill 392 https://github.com/adempiere/adempiere/releases/tag/3.9.3
Fulfill 393 https://github.com/adempiere/adempiere/releases/tag/3.9.3
Fulfill 394 https://github.com/adempiere/adempiere/releases/tag/3.9.4

This story will continue ...

**Comment by adaxa-steven** [comment]
Hi Victor,
I think that the following comment can create the wrong impression:  "Steven Sackett resigns from the foundation and creates his own fork called Adaxa Suite based on ADempiere https://en.wikipedia.org/wiki/Adaxa_Suite" 
The following may give a clearer understanding....
(1) The Adaxa Suite was created back in 2010. It was not a fork of Adempiere, it was the packaging with Adempiere of a series of other open source projects like Drupal and Asterisk for marketing reasons.
(2) I resigned as a Director of Adempiere e.V. in August 2015. The reason was the inability of Adempiere e.V. at that time to provide up-to-date information to Directors about whether it was meeting its financial obligations (probably due to no one being available to do the work, not because of any bad behaviour)
(3) On 8 October 2020 Adaxa contributed EUR330 to the foundation to assist with the costs of Adempiere trademark renewal. 
(4) Adaxa continued to make various contributions to Adempiere functionality, most recently contributions assisting a conversion to ZK8.6 in August 2022.

While Adaxa chooses to maintain its own code base (for a variety of reasons) I think the above suggests that we are still supportive of the Adempiere project even though we are not involved in the management of the project.

I hope the reorganisation of the project is very successful.

best wishes
Steven

**Reply by e-Evolution** [reply]
Hi Steven! Thank you very much for the precision.

In fact I can attest that Adaxa Suite is a friendly fork with the ADempiere project since we have mutually collaborated sharing code improvements between both projects in a respectful and friendly way , on the other hand it is true Steven and Adaxa have financially supported and made available the Adaxa Suite code for the ADempiere team to integrate its improvements to the core doing a great job, very important significant contributions such as CRM functionality and Light Production to mention just a few but not the only ones.

I also appreciate you sharing your reasons for resigning from the foundation and that this has nothing to do with the creation of the Adaxa Suite fork.

I can say that I understand perfectly, participating in the foundation has its implications

On the other hand I am very interested in knowing what they are (for a variety of reasons) to keep a fork out of ADempiere of course if you feel comfortable sharing them, I think I can intuit, it would be great to know them since today we are improving the project and we want to include all points of view including those of business consulting companies in order to know the important elements to consider for consulting companies like Adaxa.

Knowing their opinion would be a gold nugget that would greatly enrich the project https://github.com/adempiere/adempiere/discussions/4131

kind regards
Victor

**Reply by adaxa-steven** [reply]
Hi Victor,
re: "On the other hand I am very interested in knowing what they are (for a variety of reasons) to keep a fork out of ADempiere"
The following links give a good guide to where Adaxa is focussed...
https://gitlab.com/adaxa/ampere/-/wikis/home
...Motivation
...Changes
...Deprecated
regards
Steven

**Comment by rvergara** [comment]
Victor,

Thanks for the recollection

Just to give a bit more of context since my name and company are totally ignored in the History (yes, it is difficult to mention everything and everyone)

Besides having been one of the ADempiere founders and a council member, our company OFB Consulting) has provided financial support to the project for the last 3 years of about EUR 8.000 (I understand most of these funds are still available for the foundation to use). OFB also, as Adaxa did, contributed EUR 250 for the trademark renewal back in 2020. Besides those monetary contributions we have been increasingly helping the project with personnel manhours.

I am glad to see again old names here and hope we can move forward to improve the project and have again a vibrant community.

Regards

Ramiro

**Reply by e-Evolution** [reply]
Dear Ramiro!

Thank you very much for the presicion and I am going to make the clarification, the companies mentioned in the list are the ones that participated in the initiative to create the ABC association in a face-to-face meeting held at the 2009 world conference and days later they joined

OFB Consulting Ltda , Chile , Ramiro Vergara , CEO
Nickel Networks Inc, USA, Paul Aviles, CEO,

It is also worth mentioning that https://wiki.adempiere.net/History_of_ADempiere is accredited "An important response" for history Ramiro is one of those who participated in the founding of the project.
 
On the other hand, as one of the representatives of the foundation, I want to thank Ramiro and OFB Consulting for their enormous commitment as one of the main sponsors of the foundation.

kind regards
Victor
